go.mod

```diff
@@ -5,14 +5,14 @@ go 1.21
 toolchain go1.22.3
 
 require (
-	github.com/conductorone/baton-sdk v0.1.35
-	github.com/google/go-github/v62 v62.0.0
+	github.com/conductorone/baton-sdk v0.2.6
```

Comment:
Just fyi we probably want to use v0.2.7 because that fixes some issues with environment variables.